instrumentation/opentelemetry-instrumentation-aiokafka/src/opentelemetry/instrumentation/aiokafka/__init__.py

```diff
@@ -126,10 +126,10 @@ def _instrument(self, **kwargs):
         )
         wrap_function_wrapper(
             aiokafka.AIOKafkaConsumer,
-            "__anext__",
-            _wrap_anext(tracer, async_consume_hook),
+            "getone",
```

Comment:
Why are we changing the wrapped function?

Reply:
__anext__ evaluate `getone` under the hood, and `getone` can be used directly.